Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/Internal/Rectangle.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <AzQtComponents/AzQtComponentsAPI.h>
+
+#include <QWidget>
+#endif
+
+class QPaintEvent;
+
+namespace AzQtComponents::Internal
+{
+    //! A rectangle of a single color
```

Comment:
uber nit: Missing `.` at the end of the line (we always try and add these otherwise Doxygen can get confused - we do intend to turn this on before too long in future)

Reply:
Done